plugins/@grouparoo/postgres/package.json

```diff
@@ -42,6 +42,7 @@
     "@types/pg": "*",
     "actionhero": "25.0.16",
     "csv-parse": "4.15.4",
+    "fs-extra": "10.0.0",
```

Comment:
Doesn't look like this is used?

Reply:
It's used [here](https://github.com/grouparoo/grouparoo/blob/2426f644938144c9a5399388268afa105b8df4ba/plugins/%40grouparoo/postgres/__tests__/integration/cli-generate-from.ts#L2)

Reply:
whoa!  So we had just been lucky that it was brought in as a transient dependency (probably via @grouparoo/core) until now.  Good find!